extensions/extensionsmanagement.py

```diff
@@ -208,6 +225,16 @@ async def extensions(self, ctx):
         await ctx.send(ctx.author.mention, embed=embed_loaded)
         await ctx.send(ctx.author.mention, embed=embed_unloaded)
 
+    @commands.Cog.listener()
```

Comment:
This has nothing to do with managing extensions and would therefore not fit in here. Furthermore, the extension is extension management that cannot be reloaded/unloaded. This would be important to me with this feature (so it would be best if I packed my own extension.